R/use_extendr.R

```diff
@@ -152,16 +146,19 @@ extendr_module! {{
     )"
   )
 
-  make_example_wrappers(pkg_name, wrappers_file, extra_items = example_function_wrapper)
+  make_example_wrappers(pkg_name, wrappers_file, extra_items = example_function_wrapper, path = path)
+  write_namespace(pkg_name, path, quiet = quiet)
```

Comment:
I don't like that we're modifying the namespace file. In particular since we're then telling the user to run `rextendr::document` for the changes to take effect. Wouldn't that call generate the namespace file? Is there something I'm missing? In either case, let's leave modifying the namespace file to `devtools::document()`. We can call that if necessary.

Reply:
As I said above, before `rextendr::document`, it was helping with the first-time wrapper generation.